packages/angular_devkit/build_angular/src/webpack/configs/stats.ts

```diff
@@ -41,6 +41,12 @@ const verboseWebpackOutputOptions = {
   logging: 'verbose',
 };
 
+if (!isWebpackFiveOrHigher()) {
```

Comment:
Nit: the not/! here can be removed if the logic below is inverted.

```
if (isWebpackFiveOrHigher()) {
  verboseWebpackOutputOptions['modulesSpace'] = Infinity;
} else {
  verboseWebpackOutputOptions['maxModules'] = Infinity;
}
```

Reply:
done